app/policies/timelog_policy.rb

```diff
@@ -7,23 +7,23 @@ def initialize(current_user, timelog)
   end
 
   def index?
-    current_user
+    @current_user.admin?
   end
 
   def create?
-    current_user
+    @current_user.admin?
   end
 
   def update?
-    current_user.admin? || (timelog.try(:user_id) == current_user.id) 
+    @current_user.admin? || (@timelog.try(:user_id) == @current_user.id)
   end
 
   def destroy?
-    current_user.admin? || (timelog.try(:user_id) == current_user.id)
+    @current_user.admin? || (@timelog.try(:user_id) == @current_user.id)
```

Comment:
this is minor, but you don't need to execute `try` here. It is not possible to have `timelog` without `user`. If it is, we should review schema and add foreign key restriction and model validation.